src/utils/rewriteImports.js

```diff
@@ -17,7 +17,8 @@ export default (error, file, contents, moduleContext, callback) => {
   const rewritten = contents.replace(importRegexp, (entire, single, double, unquoted) => {
     const oldImportPath = single || double || unquoted;
     const absoluteImportPath = path.join(path.dirname(file), oldImportPath);
-    const newImportPath = path.relative(moduleContext, absoluteImportPath);
+    const relImportPath = path.relative(moduleContext, absoluteImportPath);
+    const newImportPath = relImportPath.split(path.sep).join('/');
```

Comment:
@justin808 I think this could be done recursively with path.parse():

    // works on Mac - NOT throughly tested against potential paths or Windows / Linux
    function splitPath(pathStr) {
        const {root, dir, base} = path.parse(pathStr);
	
        if (dir !== root) {
		// recurse
		return splitPath(dir).concat([base]);
        } else {
		// exit condition of recursion
		if (dir === '') {
			return [];
		} else {
			// so that paths which start with separator 
			// would reconstitute to start with separator
			return [''];
            }
        }
    }

    // later
    const newImportPath = splitPath(relImportPath).join('/')


I don't see any other function in NodeJS that would accomplish what you are describing more directly.